internal/plugin6/validation/write_only.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) The OpenTofu Authors
```

Comment:
As this is the same as the one in "./internal/plugin", can you add a comment saying that they should be kept in sync in both files please?

(Or abstract out the logic to a central location)